.github/workflows/python.yml

```
@@ -67,12 +67,12 @@ jobs:
             image: conda-python
             title: AMD64 Conda Python 3.8 Without Pandas
             python: 3.8
-          - name: conda-python-3.7-pandas-0.23
+          - name: conda-python-3.7-pandas-1.0.0
             cache: conda-python-3.7
             image: conda-python-pandas
-            title: AMD64 Conda Python 3.7 Pandas 0.23
+            title: AMD64 Conda Python 3.7 Pandas 1.0.0
             python: 3.7
-            pandas: 0.23
+            pandas: 1.0.0
```

```suggestion
            pandas: 1.0
```

I think it is fine to test with the latest bug-fix release in the 1.0.x series